### ISSUE TITLE
NVDA focus is moving to the hidden elements and announcing unnecessary heading and grouping button information in browse mode.

### ISSUE THREAD

**Issue by Kirank915** [microsoft/BotFramework-WebChat#5128]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

Users who rely on screen readers are getting impacted if NVDA focus is moving to the hidden elements and announcing unnecessary heading and grouping button information  in browse mode.

### What browsers and screen readers do this issue affect?

Others

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### What is the public URL for the website?

https://airtuserexpdev.azurewebsites.net/Dashboard 

### How to reproduce the issue?

Step 1: Open the above URL in Edge Browser.
Step 2: AIRT Dashboard page will appear, Navigate to “Chat with Sage” chat box at the bottom of the page.
Step 3: Type question "What is accessibility process".
Step 4: Verify whether the screen reader NVDA is moving to hidden controls in browse mode or not.

### What do you expect?

NVDA focus should not move to the hidden elements and should not announce unnecessary heading and grouping button information in browse mode.

### What actually happened?

NVDA focus is moving to the hidden elements and announcing unnecessary heading and grouping button information in browse mode.
NVDA announces as 
1.'grouping what is the accessibility process button collapsed.'
2.'heading level 4 what is the accessibility process'

### Do you have any screenshots or recordings to repro the issue?

![MAS 1 3 2_Keyboard focus is moving onto hidden elements](https://github.com/microsoft/BotFramework-WebChat/assets/117266902/81464225-0b27-4c30-bc68-220f836354eb)
![MAS 1 3 2_Keyboard focus is moving onto hidden elements_headings](https://github.com/microsoft/BotFramework-WebChat/assets/117266902/5b3f5980-a2cd-479e-aab5-adb2fc79ecbf)

### MAS reference

https://aka.ms/MAS1.3.2

### WCAG reference

https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence.html

**Comment by Kirank915** [comment]
Bug copied from MSD: [BUG 11645726](https://microsoftit.visualstudio.com/OneITVSO/_workitems/edit/11645726)

**Comment by Kirank915** [comment]
#A11yMAS;#A11ySev2;#MAS1.3.2;#HCL;#Accessibility;#FeatureBotframeworkwebchat-Apr23;#MSD:Bug11645726;

**Comment by OEvgeny** [comment]
Teams doesn't use WebChat, unfortunately, so this is Microsoft Teams issue, not WebChat issue.
